easybuild/easyblocks/p/psmpi.py

```diff
@@ -0,0 +1,53 @@
+##
+# Copyright 2016-2016 Ghent University, Forschungszentrum Juelich
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing the ParaStationMPI library, implemented as an easyblock
+
+@author: Damian Alvarez (Forschungszentrum Juelich)
+"""
+
+from easybuild.easyblocks.mpich import EB_MPICH
+
+
+class EB_psmpi(EB_MPICH):
+    """
+    Support for building the ParaStationMPI library.
+    - some compiler dependent configure options
+    """
+
+    def add_default_options(self):
+        """
+        Skip common options found in other MPICH-based runtimes
+        """
+        pass
```

Comment:
I actually like it better like this, since it's very explicit.

Maybe replace the `pass` with `self.log.info("Not using MPICH-specific configure options to configure psmpi")